guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+
+There are a few different types of wallets that let you send, receive, and store bitcoin.
+
+{% include /tip/open.html label="Tip: Break onboarding up" icon="info" color="blue" %}
+
+The wallet creation process is an excellent place to break things up into different activities, events, and tasks. For example, you might want to dedicate multiple screens when generating a [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) to explain why it is essential.
+
+{% include /tip/close.html %}
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen@2x.png"
+   width = 250
+   height = 250
+   alt-text = "Onboarding welcome screen"
+   layout = "float-left-desktop -background -shadow"
+%}
+
+After downloading a wallet application, users are typically presented with the option of either creating a new wallet or restoring an existing one. Obviously, people who have not used bitcoin before will be going through the flow of creating a new wallet.
+
+Once an individual selects “create a new wallet”, it’s typical for beginner-friendly applications to first present informational carousels that provide a high-level overview of the implications, benefits, and responsibility that comes with having a [non-custodial]({{ '/guide/glossary/#non-custodial--custodial-wallet' | relative_url }}) wallet. This moment is a great place to explain that, unlike traditional centralized financial products, a user’s “account” information cannot be recovered by the respective company. Rather, it is up to the user to safely protect their wallet information.
+
+</div>
+
+{% include /tip/open.html label="Don’t hide the implications of a non-custodial wallet" icon="forbid" color="red" %}
+
+Within your onboarding process, be clear about what it means to have a non-custodial wallet and the responsibility that they must accept. Within your onboarding process, be clear that your respective team cannot recover "account" information.
+
+{% include /tip/close.html %}
+
+While this may sound daunting to first-time users, these caveats mustn't be hidden. Doing so may severely compromise their funds' safety, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame these Bitcoin-specific features, users will feel well-equipped to confidently navigate and use your wallet.
```

Comment:
```suggestion
While this may sound daunting to first-time users, these caveats mustn't be hidden. Doing so may severely compromise their funds' safety, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame the features of self-custody bitcoin wallets, users will feel well-equipped to confidently navigate and use your wallet.
```

Reply:
Important to keep emphasizing self-custody principle. Not all bitcoin wallets are built the same